samples/boards/arc_secure_services/README.rst

```diff
@@ -0,0 +1,17 @@
+.. _arc_secure_service:
```

Comment:
Ping me when there's documentation to review...

Reply:
pls review the documentation.